README.md

```diff
@@ -1,9 +1,29 @@
-# Project Hub
+# Grant Hub
```

Comment:
Nit: while we are at it, we should update this also in `client/README.md`

Reply:
Good call!